.bazelrc.common

```diff
@@ -42,3 +42,6 @@ build:openbsd --nolegacy_external_runfiles
 # on the host.
 # See https://github.com/angular/angular/issues/27514.
 build --incompatible_strict_action_env
+
+# TODO: enable once this is supported in Bazel
+# build --experimental_allow_unresolved_symlinks
```

Comment:
I guess we need a separate CI job to test this?

Reply:
Yea, and it would anyway right now since it is not yet supported by Bazel in a way that works with rules_js even without RBE in the 6.0.0 pre-release

Reply:
So the change to support it hasn't even been published in a pre-release yet?

Reply:
Not yet. There are a number of fixes needed. One landed and the rest are in PRs.